tests/test/snapshot/test_snapshot_client_server.cpp

```diff
@@ -124,6 +126,169 @@ TEST_CASE_METHOD(SnapshotClientServerFixture,
     deallocatePages(snap.data, 5);
 }
 
+TEST_CASE_METHOD(SnapshotClientServerFixture,
+                 "Test detailed snapshot diffs with merge ops",
+                 "[snapshot]")
+{
+    // Set up a snapshot
+    std::string snapKey = std::to_string(faabric::util::generateGid());
+    faabric::util::SnapshotData snap = takeSnapshot(snapKey, 5, false);
+
+    // Set up a couple of ints in the snapshot
+    int offsetA1 = 5;
```

Comment:
As mentioned earlier, could we have a test where one diff is at offset `x` and the other one at offset `x + sizeof(int)`?